en/prepare/picture_path.rst

```diff
@@ -0,0 +1,28 @@
+
+Configurable picture path
+=========================
+
+QFieldSync gives the possibility to configure the path of the attachments.
```

Comment:
```suggestion
QFieldSync gives the possibility to configure the path of picture attachments.
```

Let's be consistent in here (picture, photo, attachments - all refers to the same)